core/templates/domain/topic/topic-update.service.ts

```diff
@@ -434,7 +434,7 @@ angular.module('oppia').factory('TopicUpdateService', [
           topic, subtopicId, thumbnailFilename) {
         var subtopic = topic.getSubtopicById(subtopicId);
         if (!subtopic) {
-          throw Error('Subtopic doesn\'t exist');
+          throw new Error('Subtopic doesn\'t exist');
```

Comment:
Weird these changes are already in develop. Can you try merging from develop? Thanks!

Reply:
Done.